bitergia.tf

```diff
@@ -293,3 +293,54 @@ resource "aws_elb" "bitergia-elb" {
     project                   = "metrics"
   }
 }
+
+resource "aws_elasticsearch_domain" "bitergia-rust-metrics-es" {
+    provider                          = "aws.us-west-1"
+    domain_name                       = "bitergia-rust-metrics-es"
+    elasticsearch_version             = "2.3"
+
+    access_policies                   = <<CONFIG
+{
```

Comment:
nit: I was thinking that its more readable to define this kind of multi line strings as variables.